tests/time_tests/test_runner/conftest.py

```diff
@@ -96,3 +107,43 @@ def get_dict_values(d):
     values = list(get_dict_values(val))
 
     return "-".join(["_".join([key, val]) for key, val in zip(keys, values)])
+
+
+@pytest.mark.hookwrapper
+def pytest_runtest_makereport(item, call):
+    """Pytest hook for report preparation.
+
+    Report tests' data to DB.
+    """
+
+    KEY_FIELDS = ('test_name', 'executable', 'model', 'device', 'niter',
+                  #'build_url'
```

Comment:
add

Reply:
rename to `run_id`. Add description that it's some unique mark and may contain some url

Reply:
Done